spec/lib/myfinance/resources/receivable_account_spec.rb

```diff
@@ -161,4 +230,79 @@
       end
     end
   end
+
+  describe "#create_as_recurrent", vcr: true do
+   let(:params) { { due_date: '2015-10-20', amount: 150.99, create_as_recurrent: "monthly" } }
+   subject { client.receivable_accounts.create(entity_id, params) }
+
+   context "with success" do
+     it "creates a new object" do
+       expect(subject.id).to eq(1298917)
+       expect(subject.due_date).to eq(Date.new(2015, 10, 20))
+       expect(subject.entity_id).to eq(entity_id)
+       expect(subject.status).to eq(1)
+       expect(subject.status_name).to eq('unreceived')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.